common/src/main/kotlin/systems/ajax/application/port/out/WordsDeletingFromLibraryRepository.kt

```diff
@@ -0,0 +1,7 @@
+package systems.ajax.application.port.out
+
+import reactor.core.publisher.Mono
+
+interface  WordsDeletingFromLibraryRepository {
```

Comment:
looks like this shouldn't be in a common module. Move it to subdomain module please

Reply:
my bad. It is my solution of cyclic dependency, because i library-module needs deleting all words from library (it is implemented in word-module), and word-module needs find libraryId by libraryName and userId(it is implemented in library-module). I relocate method of deleting words from library to this interface. 
I placed the interface in library-module, would it be ok?

Reply:
```suggestion
interface WordsDeletingFromLibraryRepository
```